srcpkgs/zeroc-mcpp/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'zeroc-mcpp'
+pkgname=zeroc-mcpp
+version=2.7.2.14
+revision=1
+wrksrc="mcpp-${version}"
+build_style=gnu-makefile
+short_desc="Simplified fork of mcpp from ZeroC, for building ICE"
+maintainer="a dinosaur <nick@a-dinosaur.com>"
+license="BSD-2-Clause"
+homepage="https://github.com/zeroc-ice/mcpp"
+distfiles="https://github.com/zeroc-ice/mcpp/archive/v${version}.tar.gz"
+checksum=1a426cf2d513ffd66634384d5445dcce1aac83be1789a56044013eab8ca8ffff
+nostrip=yes
+conflicts=mcpp-devel
+
+post_patch() {
+	vsed 's|LIBDIR = lib64|LIBDIR = lib|' -i Makefile
+}
+
+do_install() {
+	vinstall mcpp_lib.h 644 usr/include
+	vinstall mcpp_out.h 644 usr/include
+	vinstall lib/libmcpp.a 644 usr/lib
+	vlicense LICENSE
+}
```

Comment:
```suggestion
post_install() {
	vinstall mcpp_lib.h 644 usr/include
	vinstall mcpp_out.h 644 usr/include
	vlicense LICENSE
}
```

This is a nitpick, so you don't need to apply it. But I think this works. And from what I looked at the Makefile, you only really need to set `LIBDIR` if you're using `make install`.

Reply:
Ditto.